### ISSUE TITLE
Phishing wallet-19Y4NSiG8kkzwWjMD17euEaQ5PErpwxWkP

### ISSUE THREAD

**Issue by sunilkumar4022** [OmniLayer/omniexplorer#134]
Hi, 

My Tether transaction has been compromised and all of my Tether - USDT were transferred to the recipient-19Y4NSiG8kkzwWjMD17euEaQ5PErpwxWkP which I don't even know the source.

I do remember that my recipient was not the same but some how the transaction has been redirected to it. I request you to kindly allot me the details of address and report its source.

Additionally, there are few more transaction to same recipient and have lost their funds.

Thank you

TX ID-https://omniexplorer.info/tx/e9df12c01bc4e2bee7af728223e498235bba725750452c9d7af3cfe7ca153e35

**Comment by achamely** [comment]
@sunilkumar4022 

Unfortunately we do not know the details of any addresses or their source. 
However a quick google search shows that address has been in some public discussions

It appears bitcoin talk member mworker may claim it as their own address:
Source:   https://bitcointalk.org/index.php?topic=2195541.msg22176779#msg22176779
So you may try reaching out to them (they have their twitter/facebook info in that post as well). 

Additionally searches show that address being reported as part of a clipboard virus that replaces any bitcoin address in your clipboard with itself:
Source: 
https://forum.bitcoin.com/alternative-clients/error-transfer-t17788.html
https://bitcointalk.org/index.php?topic=1960695.msg19488391#msg19488391

At the very minimum you should have your computer checked for malware/virus and then try contacting the mworker handle and see what they say if anything

**Comment by sunilkumar4022** [comment]
Hi,

I have learnt about it through the links that my laptop was injected with
malware and the recipient address was changed during my transaction. Thank
you for your support.

Kindly assist me with little aspect as of how did the cross transaction did
succeeded? Its quite obvious that the transaction recipient is of Bitcoin
address but I have transferred Tether so how come the transaction is
success?

I'm certain that it is cross block chain transaction but how did it
succeeded? Is it because of Tether inability to realize its source or it
may be ineffective during these type of cross block chaining?

With Regards,
Sunil Kumar,
+91-9676719142.
